src/pages/project/project_wizard/RelatedRegistrationsField.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material';
+import { useFormikContext } from 'formik';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchRegistrationsForProject } from '../../../api/hooks/useFetchRegistrationsForProject';
+import { ListPagination } from '../../../components/ListPagination';
+import { ListSkeleton } from '../../../components/ListSkeleton';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { CristinProject } from '../../../types/project.types';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+
+export const RelatedRegistrationsField = () => {
+  const { t } = useTranslation();
+  const { values } = useFormikContext<CristinProject>();
+  const [page, setPage] = useState(1);
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+
+  const resultsQuery = useFetchRegistrationsForProject(values.id, rowsPerPage, page);
```

Comment:
Samme som i `ProjectResultsAccordion`, at vi kunne brukt eksisterende `useRegistrationSearch` vi har fra før for å søke opp relaterte resultat